kalite/distributed/static/js/distributed/exercises/api_resources.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.http import HttpResponse
+
+from tastypie import fields
+from tastypie.resources import ModelResource
```

Comment:
Please follow the import structure outlined [here](https://github.com/learningequality/ka-lite/wiki/Coding-guidelines-and-conventions#imports)